client/app.json

```diff
@@ -1,5 +1,7 @@
 {
   "pages":[
+    "pages/home/home",
```

Comment:
Should we create a new home page or just modify the existing index page?

Reply:
I am not sure whether there is a convention just leave the index page there. If not I think we can just modify the index page.

Reply:
Please do it. It sounds more reasonable to me :)

Reply:
Okay, no problem!

Reply:
Oh, do I need to delete the unrelated page such as "addCgi" page?

Reply:
Yes please.

Reply:
The page has been deleted, please check that.

Reply:
Although you've added `.DS_Store` to `.gitignore`, the files that you have already added won't be removed automatically. Please run `git rm *.DS_Store` and commit.

Reply:
Deleted, please check.